llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -946,8 +946,11 @@ static Error materializeCUDie(DWARFCompileUnit &DCU,
   DWARFDataExtractor DWARFExtractor(CUData, DCU.isLittleEndian(),
                                     DCU.getAddressByteSize());
   uint64_t PrevOffset = CUOffset;
-  uint64_t AbbrevTag = DWARFExtractor.getULEB128(&CUOffset);
-  std::memcpy(&SectionContents[SectionOffset], &AbbrevTag,
+  Error Err = Error::success();
+  uint64_t AbbrevTag = DWARFExtractor.getULEB128(&CUOffset, &Err);
+  if (Err)
+    return Err;
+  std::memcpy(&SectionContents[SectionOffset], &CUData.data()[PrevOffset],
```

Comment:
Nit: if `&CUData.data()[PrevOffset]` is equivalent to `CUData.data() + PrevOffset` that might be easier to read.